.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to be released'
+        required: true
+      prerelease:
+        description: "Is the release a GA? (true / false)"
```

Comment:
The description "is it a GA?" seems to be the opposite of the section name "is it a prerelease?", which one is right?